code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm

```diff
@@ -136,7 +136,7 @@
 	/// The maximum amount of vines a plant can have at one time
 	var/max_vines = 4
 	/// How far away a plant can attach a vine to something
-	var/vine_grab_distance = 4 //SKYRAT EDIT - Original 5
```

Comment:
Rather see incremental buffs than a bunch at once, and this is what I'd leave out